### PR TITLE
[zh-cn] sync pod-security-standards.md user-namespaces.md sysctl-cluster.md

### PR DESCRIPTION
**Pull request by hunshcn**

### CODE REVIEW

**Review by windsonsea — COMMENTED** [review]
It's required to include some in-synced content in #44156 for `pod-security-standards.md` in this PR. 

Or you can find these differences by running this command on your `main` branch:

```
./scripts/lsync.sh content/zh-cn/docs/concepts/security/pod-security-standards.md
```

**Comment by hunshcn** [comment]
@windsonsea done

File: content/zh-cn/docs/concepts/security/pod-security-standards.md [hunk 1]
```diff
@@ -594,6 +598,21 @@ Restrictions on the following controls are only required if `.spec.os.name` is n
...
+isolation. How they work together with Pod Security Standards is described in
+the [documentation](/docs/concepts/workloads/pods/user-namespaces#integration-with-pod-security-admission-checks) for Pods that use user namespaces.
+-->
+
+## 用户命名空间
+
+用户命名空间是 Linux 特有的功能，可在运行工作负载时提高隔离度。
+关于它们如何与 Pod Security Standard 协同工作，请参阅
+[文档](/zh-cn/docs/concepts/workloads/pods/user-namespaces#integration-with-pod-security-admission-checks)
+中为使用用户命名空间的 Pod 描述。
```
Comment (windsonsea) [inline content/zh-cn/docs/concepts/security/pod-security-standards.md:614]:
```suggestion
了解 Pod 如何使用用户命名空间。
```

**Comment by windsonsea** [comment]
Please [squash your commits](https://kubernetes.io/docs/contribute/new-content/open-a-pr/#squashing-commits) into a single one.

File: content/zh-cn/docs/concepts/workloads/pods/user-namespaces.md [hunk 2]
```diff
@@ -279,6 +279,60 @@ Pod 的 UID/GID 不会与主机的文件所有者/组相匹配。
 
 [CVE-2021-25741]: https://github.com/kubernetes/kubernetes/issues/104980
 
+<!--
+## Integration with Pod security admission checks
```
[the remark this answers is not in the document]
Comment (hunshcn) [reply content/zh-cn/docs/concepts/workloads/pods/user-namespaces.md:283]: 确实没太注意，已解决，麻烦了

File: content/zh-cn/docs/concepts/workloads/pods/user-namespaces.md [hunk 3]
```diff
@@ -279,6 +279,53 @@ Pod 的 UID/GID 不会与主机的文件所有者/组相匹配。
...
+This behavior can be controlled by the [feature
+gate](/docs/reference/command-line-tools-reference/feature-gates/)
+`UserNamespacesPodSecurityStandards`, which allows an early opt-in for end
+users. Admins have to ensure that user namespaces are enabled by all nodes
+within the cluster if using the feature gate.
+-->
+
+对于启用了用户命名空间的 Linux Pod，Kubernetes 会以受控方式放宽
+[Pod 安全性标准](/zh-cn/docs/concepts/security/pod-security-standards) 的应用。
+这种行为可以通过 [特性门控](/zh-cn/docs/reference/command-line-tools-reference/feature-gates/)
```
Comment (windsonsea) [inline content/zh-cn/docs/concepts/workloads/pods/user-namespaces.md:302]:
```suggestion
这种行为可以通过[特性门控](/zh-cn/docs/reference/command-line-tools-reference/feature-gates/) 
```

File: content/zh-cn/docs/concepts/workloads/pods/user-namespaces.md [hunk 4]
```diff
@@ -279,6 +279,53 @@ Pod 的 UID/GID 不会与主机的文件所有者/组相匹配。
...
+gate](/docs/reference/command-line-tools-reference/feature-gates/)
+`UserNamespacesPodSecurityStandards`, which allows an early opt-in for end
+users. Admins have to ensure that user namespaces are enabled by all nodes
+within the cluster if using the feature gate.
+-->
+
+对于启用了用户命名空间的 Linux Pod，Kubernetes 会以受控方式放宽
+[Pod 安全性标准](/zh-cn/docs/concepts/security/pod-security-standards) 的应用。
+这种行为可以通过 [特性门控](/zh-cn/docs/reference/command-line-tools-reference/feature-gates/) 
+`UserNamespacesPodSecurityStandards` 进行控制， 终端用户可以选择开启它。
```
Comment (windsonsea) [inline content/zh-cn/docs/concepts/workloads/pods/user-namespaces.md:303]:
```suggestion
`UserNamespacesPodSecurityStandards` 进行控制，此特性可以让终端用户提前选择。
```

File: content/zh-cn/docs/concepts/workloads/pods/user-namespaces.md [hunk 5]
```diff
@@ -279,6 +279,53 @@ Pod 的 UID/GID 不会与主机的文件所有者/组相匹配。
...
+如果使用特性门控，管理员必须确保群集中的所有节点都启用了用户命名空间。
```
Comment (windsonsea) [inline content/zh-cn/docs/concepts/workloads/pods/user-namespaces.md:304]:
```suggestion
如果管理使用此特性门控，必须确保群集中的所有节点都启用了用户命名空间。
```

File: content/zh-cn/docs/concepts/workloads/pods/user-namespaces.md [hunk 6]
```diff
@@ -279,6 +279,53 @@ Pod 的 UID/GID 不会与主机的文件所有者/组相匹配。
...
+<!--
+For Linux Pods that enable user namespaces, Kubernetes relaxes the application of
+[Pod Security Standards](/docs/concepts/security/pod-security-standards) in a controlled way.
+This behavior can be controlled by the [feature
+gate](/docs/reference/command-line-tools-reference/feature-gates/)
+`UserNamespacesPodSecurityStandards`, which allows an early opt-in for end
+users. Admins have to ensure that user namespaces are enabled by all nodes
+within the cluster if using the feature gate.
+-->
+
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/workloads/pods/user-namespaces.md:299]: 299 多余空行

File: content/zh-cn/docs/concepts/workloads/pods/user-namespaces.md [hunk 7]
```diff
@@ -279,6 +279,53 @@ Pod 的 UID/GID 不会与主机的文件所有者/组相匹配。
...
+对于启用了用户命名空间的 Linux Pod，Kubernetes 会以受控方式放宽
+[Pod 安全性标准](/zh-cn/docs/concepts/security/pod-security-standards) 的应用。
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/workloads/pods/user-namespaces.md:301]: 按风格指南要求，中文之间不要留白。

```suggestion
[Pod 安全性标准](/zh-cn/docs/concepts/security/pod-security-standards)的应用。
```

File: content/zh-cn/docs/concepts/workloads/pods/user-namespaces.md [hunk 8]
```diff
@@ -279,6 +279,53 @@ Pod 的 UID/GID 不会与主机的文件所有者/组相匹配。
...
+这种行为可以通过[特性门控](/zh-cn/docs/reference/command-line-tools-reference/feature-gates/) 
+`UserNamespacesPodSecurityStandards` 进行控制，此特性可以让终端用户提前选择。
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/workloads/pods/user-namespaces.md:303]:
```suggestion
`UserNamespacesPodSecurityStandards` 进行控制，可以让最终用户提前尝试此特性。
```

File: content/zh-cn/docs/concepts/workloads/pods/user-namespaces.md [hunk 9]
```diff
@@ -279,6 +279,53 @@ Pod 的 UID/GID 不会与主机的文件所有者/组相匹配。
...
+如果管理使用此特性门控，必须确保群集中的所有节点都启用了用户命名空间。
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/workloads/pods/user-namespaces.md:304]:
```suggestion
如果管理员启用此特性门控，必须确保群集中的所有节点都启用了用户命名空间。
```

File: content/zh-cn/docs/concepts/security/pod-security-standards.md [hunk 10]
```diff
@@ -594,6 +598,21 @@ Restrictions on the following controls are only required if `.spec.os.name` is n
...
+User Namespaces are a Linux-only feature to run workloads with increased
+isolation. How they work together with Pod Security Standards is described in
+the [documentation](/docs/concepts/workloads/pods/user-namespaces#integration-with-pod-security-admission-checks) for Pods that use user namespaces.
+-->
+## 用户命名空间    {#user-namespaces}
+
+用户命名空间是 Linux 特有的功能，可在运行工作负载时提高隔离度。
+关于用户命名空间如何与 PodSecurityStandard 协同工作，
+请参阅
+[文档](/zh-cn/docs/concepts/workloads/pods/user-namespaces#integration-with-pod-security-admission-checks)
```
Comment (windsonsea) [inline content/zh-cn/docs/concepts/security/pod-security-standards.md:613]:
```suggestion
请参阅[文档](/zh-cn/docs/concepts/workloads/pods/user-namespaces#integration-with-pod-security-admission-checks)
```
渲染后，阅文 是2个汉字，汉字之间不要折行

**Review by windsonsea — COMMENTED** [review]
nit
/approve

**Review by ydFu — APPROVED** [review]
/lgtm

**Pull request merged**